.github/workflows/grapl-build.yml

```diff
@@ -100,8 +101,15 @@ jobs:
 
   build-zips:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
     steps:
       - uses: actions/checkout@v2
```

Comment:
Do the "set up python" after the "clean GH actions space" - that's why it cant find 3.7

Reply:
aha! thanks. Fixed.